RiotSwiftUI/Modules/UserSessions/UserOtherSessions/UserOtherSessionsModels.swift

```diff
@@ -32,25 +32,22 @@ enum UserOtherSessionsViewModelResult: Equatable {
 
 struct UserOtherSessionsViewState: BindableState, Equatable {
     var bindings: UserOtherSessionsBindings
-    let title: String
-    var sections: [UserOtherSessionsSection]
+    var title: String
+    var items: [UserSessionListItemViewData]
```

Comment:
I would avoid the use of `items` and instead refer to `sessions` to be more explicit

Reply:
Agree 👍  I will rename it to `sessionItems` as we reserved `session` for `MXSession`